src/boot/boot.js

```diff
@@ -63,12 +63,16 @@ function injectStylesheet(doc, href) {
 /**
  * @param {Document} doc
  * @param {string} src - The script URL
- * @param {boolean} forceReload - Whether to force re-evaluation of the script,
- *   in case it has been loaded in the same document previously.
+ * @param {object} options
+ *   @param {boolean} [options.esModule] - Whether to load the script as an ES module
+ *   @param {boolean} [options.forceReload] - Whether to force re-evaluation of an ES module script
```

Comment:
I'm not seeing this option actually used (i.e. set to `true`) anywhere anymore? Is it still relevant?